app/pages/changeActivity/OperatePackagingSiteOwnBrandsPage.scala

```diff
@@ -29,4 +30,11 @@ case object OperatePackagingSiteOwnBrandsPage extends QuestionPage[Boolean] {
   override def toString: String = "operatePackagingSiteOwnBrands"
 
   override val url: Mode => String = mode => routes.OperatePackagingSiteOwnBrandsController.onPageLoad(mode).url
+
+  override val redirectConditions: UserAnswers => List[RequiredPage] = userAnswers => {
+    List(
+      RequiredPage(AmountProducedPage),
+      RequiredPage(ThirdPartyPackagersPage, additionalPreconditions = List(userAnswers.get(AmountProducedPage).contains(AmountProduced.Small)))
+    )
+  }
```

Comment:
I believe this page is also required for a large producer.  It only does not show up for None.

Reply:
Yes but the TPP packagers page is only required for SmallProducer?

See, for example, Line 41 in app/pages/changeActivity/ImportsPage.scala